File: Revisao_Repeticao/operadores_repeticao.py
n = 10
total = sum(range(1, n+1))
print("sum of 1..10 =", total)
a,b = 0,1
fibs = []
for _ in range(10):
    fibs.append(a)
    a,b = b, a+b
print(fibs)

File: Revisao_Repeticao/operadores_repeticao.cpp
#include <iostream>
using namespace std;
int main() {
    int n = 10, total = 0;
    for (int i = 1; i <= n; ++i) total += i;
    cout << "sum of 1..10 = " << total << "\n";
    int a = 0, b = 1;
    cout << "fibs: ";
    for (int i = 0; i < 10; ++i) {
        cout << a << (i<9 ? ", " : "\n");
        int next = a + b; a = b; b = next;
    }
    return 0;
}